securesignals-ima-devapp/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.uid2.securesignals.ima.devapp'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "com.uid2.securesignals.ima.devapp"
+        minSdk 26
+        targetSdk 33
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
```

Comment:
A number of these options are actually duplicates across all the different projects. That's why I added `common.gradle` so that we had a centralised copy of them and we could keep everything in sync. Check `dev-app/build.gradle` to compare and also this line:

```
apply from:  rootProject.file("$rootDir/common.gradle")
```

Except for some additional `dependencies`, I would expect this file to be very nearly 1:1 with `dev-app/build.gradle`